front-end-tools/src/constants.ts

```diff
@@ -4,3 +4,14 @@ import moment from 'moment';
 export const REFRESH_INTERVAL = moment.duration(5, 'seconds');
 
 export const BROWSER_WALLET = ephemeralConnectorType(BrowserWalletConnector.create);
+
+export const DEFAULT_JSON_OBJECT = {
+    myStringField: 'FieldValue',
+    myNumberField: 4,
+    myArray: [1, 2, 3],
+    myObject: {
+        myField1: 'FieldValue',
+    },
+};
+
+export const DEFAULT_ARRAY_OBJECT = 'Examples: \n\n[1,2,3] or \n\n["abc","def"] or \n\n[{"myFieldKey":"myFieldValue"}]';
```

Comment:
This is an oddly named constant, if I compare it to DEFAULT_JSON_OBJECT. This is a string. Could you document the two please.

Reply:
Added the following comment:

// This is the example array that is shown in the input parameter textarea as a placeholder when the user has no embedded schema in the module 
// or does not want to use the embedded schema (meaning if the checkbox "Use module from step 1" is unchecked).

Reply:
Renamed it to:
`EXAMPLE_ARRAYS`